### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by macolo** [callowayproject/django-categories#102]
I understand that there is currently no python 3 support in django-categories.

This is probably blocked by /django-mptt/django-mptt#395

**Comment by coordt** [comment]
@macolo I've been thinking, django-mptt is causing a bunch of trouble lately. Should we switch to something else?

**Comment by macolo** [comment]
@coordt I don't think there is anything else, after some simple google research. Maybe there should be some investigation into why category management is not in demand anymore. Are there maybe simpler, native solutions that became viable through improvements of the django admin interface?

**Comment by macolo** [comment]
/django-mptt/django-mptt actually has support for python 3 they just forget to update their readme file.

So I guess this issue here is not blocked in any way.

**Comment by epicserve** [comment]
I have a project I would like to upgrade to Python 3 and django-categories is one of requirements that is blocking being able to upgrade. I'm happy to help if I can find time.

**Comment by WarmongeR1** [comment]
+1

**Comment by ZhiweiWang** [comment]
+1 too

**Comment by toabi** [comment]
+1

**Comment by gagandeep** [comment]
+1

**Comment by epicserve** [comment]
For what it's worth I fixed everything I needed to fix for my work project to work with Python 3. I haven't taken any time to make the code so it's backwards compatible. I'm willing to work on it some more if it's possible that it will get merged in.

https://github.com/epicserve/django-categories/tree/py3

@coordt, let me know if you want me to work on it some more and if you think you could merge it in once backwards compatibility with Python 2 is added.

**Comment by coordt** [comment]
@epicserve I just made you a contributor to the project. I'm having difficulties keeping up with things, so I would really love it if you could merge it in. We will need it to be py2 compatible, if that isn't too much to ask.

**Comment by epicserve** [comment]
@coordt I've spent some time getting things compatible, however when I run tests locally using tox everything passes, but they don't pass when using Travis for Python 2.7. Any ideas, here is the Travis build I'm talking about.

https://travis-ci.org/callowayproject/django-categories/jobs/109434427

**Comment by epicserve** [comment]
@coordt Never mind, got it working. Closing this issue now that it's merged into master from PR #108.
